torch/onnx/_internal/fx/export.py

```diff
@@ -29,7 +140,7 @@ def _create_op_overload_to_exporter_key_table() -> Dict[torch._ops.OpOverload, s
             continue
 
         exporter_look_up_key = op_overload_packet._qualified_op_name
-        if registration.registry.get_function_group(exporter_look_up_key) is None:
+        if _ATENLIB_FUNCTIONS.get(exporter_look_up_key) is None:
```

Comment:
This will reduce the coverage of current exporter. Please make sure it doesn't break GPT and bloom models.

Reply:
Draft mode rn. More tests on the way